pydrad/parse/util.py

```diff
@@ -54,6 +52,9 @@ def read_amr_file(filename):
     # NOTE: This is done after creating the table because the
     # remaining number of columns can be variable and thus we
     # cannot assign all of the column names at once.
+    if table.dtype[len(columns)-1] == np.int64:
+        columns.remove('electron_mass_density')
+        del units['electron_mass_density']
```

Comment:
Can you add a comment explaining this conditional?

Reply:
The .amr files have a combination of doubles and ints.  The columns we care about (the actual data) are the doubles, while the other columns are ints with info about the grid refinement.

Basically, if the seventh column is an int, electron mass density is **_not_** in the file.  If the seventh column is a double, then electron mass density **_is_** in the file.

Since each row can be variable length because of the refinement, this seemed like the easiest way to do it.